client/rest-high-level/src/main/java/org/elasticsearch/client/IndicesClient.java

```diff
@@ -786,33 +786,70 @@ public void validateQueryAsync(ValidateQueryRequest validateQueryRequest, Reques
     }
 
     /**
-     * Gets index templates using the Index Templates API
+     * Gets index templates which includes deprecated doc types using the Index Templates API
```

Comment:
It might be nice to elaborate a bit in this comment ("Gets index templates using the Index Templates API. The mappings will be returned in a legacy deprecated format, where the mapping definition is nested under the type name.")